src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -578,159 +403,63 @@ export class PatientViewPageStore {
+    }, []);
 
-        const queryVariants = _.uniqBy(_.map(this.mutationData.result.concat(this.uncalledMutationData.result), (mutation: Mutation) => {
-            return generateQueryVariant(mutation.gene.hugoGeneSymbol,
-                this.sampleIdToTumorType[mutation.sampleId],
-                mutation.proteinChange,
-                mutation.mutationType,
-                mutation.proteinPosStart,
-                mutation.proteinPosEnd);
-        }), "id");
-
-        return fetchOncoKbData(this.sampleIdToTumorType, queryVariants);
-    }
 
     readonly oncoKbData = remoteData<IOncoKbData>({
         await: () => [
             this.mutationData,
             this.uncalledMutationData,
             this.clinicalDataForSamples
         ],
-        invoke: async() => this.oncoKbDataInvoke()
+        invoke: async() => fetchOncoKbData(this.sampleIdToTumorType, this.mutationData, this.uncalledMutationData)
     }, ONCOKB_DEFAULT);
 
     readonly cnaOncoKbData = remoteData<IOncoKbData>({
         await: () => [
             this.discreteCNAData,
             this.clinicalDataForSamples
         ],
-        invoke: async() => {
-            if (this.discreteCNAData.result.length > 0) {
-                const queryVariants = _.uniqBy(_.map(this.discreteCNAData.result, (copyNumberData: DiscreteCopyNumberData) => {
-                    return generateQueryVariant(copyNumberData.gene.hugoGeneSymbol,
-                        this.sampleIdToTumorType[copyNumberData.sampleId],
-                        getAlterationString(copyNumberData.alteration));
-                }), "id");
-                return fetchOncoKbData(this.sampleIdToTumorType, queryVariants);
-            } else {
-                return ONCOKB_DEFAULT;
-            }
-        }
+        invoke: async() => fetchCnaOncoKbData(this.sampleIdToTumorType, this.discreteCNAData)
     }, ONCOKB_DEFAULT);
 
     readonly copyNumberCountData = remoteData<CopyNumberCount[]>({
         await: () => [
             this.discreteCNAData
         ],
-        invoke: async() => {
-            const copyNumberCountIdentifiers: CopyNumberCountIdentifier[] =
-                this.discreteCNAData.result.map((cnData: DiscreteCopyNumberData) => {
-                    return {
-                        alteration: cnData.alteration,
-                        entrezGeneId: cnData.entrezGeneId
-                    };
-                });
-
-            if (this.geneticProfileIdDiscrete.result && copyNumberCountIdentifiers.length > 0) {
-                return await internalClient.fetchCopyNumberCountsUsingPOST({
-                    geneticProfileId: this.geneticProfileIdDiscrete.result,
-                    copyNumberCountIdentifiers
-                });
-            } else {
-                return [];
-            }
-        }
+        invoke: async() => fetchCopyNumberData(this.discreteCNAData, this.geneticProfileIdDiscrete)
     }, []);
 
-    @computed get indexedHotspotData(): IHotspotData|undefined

```

Comment:
This is confusing, I didn't remove this function, but github shows it as removed here, and added back a few lines below...